### PR TITLE
Added CI and useful makefile commands.

### PR DESCRIPTION
**Pull request by bwplotka**
Signed-off-by: Bartek Plotka <bwplotka@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,9 @@
 * Define your Infrastructure (e.g Terraform, Ansible, Puppet, Chef, Kubernetes etc)
 * Define any other file that you can imagine
 
-... using simple, typed and testable Go code!
+...using simple, typed and testable Go code!
+
+_`mimic`: Mimic is a super-human with the ability to copy the powers and abilities of others._
```
Comment (domgreen) [inline README.md]: :+1:

File: Makefile [hunk 2]
```diff
@@ -1,32 +1,69 @@
...
+	$(call fetch_go_bin_version,golang.org/x/tools/cmd/goimports,$(GOIMPORTS_VERSION))
 
 $(GOJSONSCHEMA):
-ifeq (${GOBIN},)
-	@echo >&2 "GOBIN environment variable is not defined, where to put installed binaries?"; exit 1
-endif
 	@echo ">> installing gojsonschema"
-	@GO111MODULE=on GOOS= GOARCH= $(GO) install github.com/atombender/go-jsonschema/cmd/gojsonschema
\ No newline at end of file
+	@GO111MODULE=on GOOS= GOARCH= $(GO) install github.com/atombender/go-jsonschema/cmd/gojsonschema
```
Comment (domgreen) [inline Makefile]: should this be moved into a make file for the example?

Comment (domgreen) [reply Makefile]: nit tbh fix later if so

**Comment by bwplotka** [comment]
Merging to unblock CI.

**Pull request merged**